modules/shared/logic/userAuth/userAuth.ts

```diff
@@ -2,7 +2,10 @@ import cookieCutter from 'cookie-cutter';
 
 export const getUserToken = (): string => cookieCutter.get('user');
 export const setUserToken = (token: string): void => {
-  cookieCutter.set('user', token, { path: '/' });
+  cookieCutter.set('user', token, {
+    path: '/',
+    expires: new Date(Date.now() + 3600 * 1000 * 24 * 30 * 356),
```

Comment:
what about abstracting these magic numbers into one variable and called it something like ten years just to make it easier in the future to understand it.